docs/appendices/release-notes/unreleased.rst

```diff
@@ -140,3 +140,6 @@ Fixes
 - Fixed ``EXPLAIN`` plan output for queries with a ``WHERE`` clause containing
   implicit cast symbols. A possible optimization of our planner/optimizer was
   not used, resulting in different output than actually used on plan execution.
+
+- Fixed formatting of ``TIMESTAMP WITHOUT TIME ZONE`` values in Admin UI query
+  console result table, by upgrading to version 1.23.1.
```

Comment:
Would be great if we'd adjust any existing "Update Admin-UI" entry instead in future. Would avoid to rephrase this on release ;)

Reply:
I thought about it. But/also the question would be if that would qualify more as a fix than an improvement. Not sure here, happy about any guidance. Let's get it right **now**, so that I can make it a habit.

Reply:
One moment, we did not release 1.23.x with CrateDB 5.0, so we should not add this as a fix and backport. Otherwise we'll add a feature to the feature freeze 5.0 version.

Reply:
=> Discussion resolved: We will release it as a regular _change_.